src/amplitude/config.py

```diff
@@ -71,6 +72,7 @@ def __init__(self, api_key: str = None,
         self.storage_provider: StorageProvider = storage_provider
         self.opt_out: bool = False
         self.plan: Plan = plan
+        self.library_context: str = library_context
```

Comment:
Can you add the new argument to class docstring too?

Reply:
Thanks for reviewing! Will update!